docs/networking.md

```diff
@@ -9,9 +9,13 @@ help you out.
 
 
 {% include /style/icon.html type="info" %}
-For Raspberry Pi platforms with no display, you must use the wired Ethernet port
-to connect for the first time. You can set up additional connections using the
-`connmanctl` command through a remote SSH session.
+For Raspberry Pi platforms with no display, you must use the 
+wired Ethernet port to connect for the first time through a remote SSH 
+session. An alternative method is to connect the pi to an HDMI screen monitor 
+and USB keyboard and press <kbd>Alt</kbd>+<kbd>F2</kbd> at the end of
```

Comment:
Also `tty6` is guaranteed (by systemd) to have a login prompt, whereas other terminals could be used for other things like `tty1` currently is. So, <kbd>F6</kbd> might be better advice than <kbd>F2</kbd>.